From 268946630ccce306607ca4f9bebe076835f37cce Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.de>
Date: Thu, 8 Aug 2013 15:36:23 +0200
Subject: [PATCH] uvc: increase number of buffers
Git-Commit: 320a145da3aefa0fe650ac591a6919da2d83783f
Patch-Mainline: v3.12
References: bnc#822164, bnc#805804

This is necessary to make new cameras from Lite-On
work. With the old number the camera will always deliver
short frames.

Signed-off-by: Oliver Neukum <oneukum@suse.de>
---
 drivers/media/video/uvc/uvcvideo.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/video/uvc/uvcvideo.h b/drivers/media/video/uvc/uvcvideo.h
index cf2401a..62ba39a 100644
--- a/drivers/media/video/uvc/uvcvideo.h
+++ b/drivers/media/video/uvc/uvcvideo.h
@@ -189,9 +189,9 @@ struct uvc_xu_control {
 /* Number of isochronous URBs. */
 #define UVC_URBS		5
 /* Maximum number of packets per URB. */
-#define UVC_MAX_PACKETS		32
+#define UVC_MAX_PACKETS		128	
 /* Maximum number of video buffers. */
-#define UVC_MAX_VIDEO_BUFFERS	32
+#define UVC_MAX_VIDEO_BUFFERS	128
 /* Maximum status buffer size in bytes of interrupt URB. */
 #define UVC_MAX_STATUS_SIZE	16
 
-- 
1.8.3.1

